source/SkiaSharp.Build.props

```diff
@@ -47,16 +47,16 @@
 
   <!-- .NET Standard, .NET FX and .NET 6 -->
   <PropertyGroup>
-    <BasicTargetFrameworks>netstandard1.3;netstandard2.0;net462</BasicTargetFrameworks>
+    <BasicTargetFrameworks>netstandard1.3;netstandard2.0;netstandard2.1;net462;netcoreapp3.1;net5.0</BasicTargetFrameworks>
+    <BasicTargetFrameworks Condition="$(BuildingForNet6)">$(BasicTargetFrameworks);net6.0</BasicTargetFrameworks>
```

Comment:
Also building for netcoreapp3.1 and net5 by default, and on platforms that support net6, build net6.

This is not added to any NuGets, but I want to do that in a new PR so we get all the new code features.